### PR TITLE
Send no response if target not specified

### PR DESCRIPTION
**Pull request by whaley**
Relates To: #208

### CODE REVIEW

**Comment by whaley** [comment]
Pick one between this and #213

File: src/main/kotlin/javabot/model/Factoid.kt [hunk 1]
```diff
@@ -61,6 +61,11 @@ class Factoid(var name: String = "", var value: String = "", var userName: Strin
         if (name.endsWith(" $^")) {
             replaced = urlencode(camelcase(replacedValue))
         }
+
+        if (replaced.isBlank()) {
+            return null;
+        }
```
Comment (whaley) [inline src/main/kotlin/javabot/model/Factoid.kt]: This is icky, but I couldn't think of a better way do this without a large sweeping change

**Comment by evanchooly** [comment]
errors should always give a user response.

**Comment by whaley** [comment]
@evanchooly Close this one and review/merge #213 in that case.

**Pull request closed without merging**